src/client/components/charts/HeatMapChart.js

```diff
@@ -56,6 +67,7 @@ class HeatMapChart extends React.Component {
               categories: this.props.yAxisCategories,
               title: 'Genes',
               labels: {
+                enabled: this.props.yAxisLabel,
```

Comment:
should I make it so that this prop is optional? ie if unspecified we assume it to be true?